_templates/component_v4/with-prompt/one-file.ejs.t

```diff
@@ -0,0 +1,30 @@
+---
+to: "<%= !multipleFiles ? `${path}/${tagName}/${tagName}.ts` : null %>"
+---
+import Component, { HTMLFragment, html, createStyle } from '@biotope/element';
+import * as styles from './<%= tagName %>.scss';
+<% generateParts.map((part) => { %>
+interface <%= componentName %><%= part %> {}<% }) %>
+
+class <%= componentName %> extends Component< <% if(generateParts.indexOf('Props') !== -1) { %><%= componentName %>Props<% } else { %>null<% } %>, <% if(generateParts.indexOf('State') !== -1) { %><%= componentName %>State<% } else { %>null<% } %> > {
+  public static componentName = '<%= tagName %>';
+<% if(generateParts.indexOf('Props') !== -1) { %>  
+  public static attributes = [];
+
+  protected readonly defaultProps: <%= componentName %>Props = {};
+<% } %><% if(generateParts.indexOf('State') !== -1) { %>   
+  protected readonly defaultState: <%= componentName %>State = {};
+<% } %><% if(generateParts.indexOf('Methods') !== -1) { %>
+  public methods: <%= componentName %>Methods = {};
+<% } %>
+  public render(): HTMLFragment {
+    return html`
+      <slot/>
+      ${createStyle(styles)}
+    `
+  }
+}
+
+<%= componentName %>.register();
+
+export default <%= componentName %>;
```

Comment:
missing EOF new line - QG will complain

Reply:
lets stop manual code style nitpicking, either we integrate QG or we ll leave them out, manual code style code review comments are unproductive